.github/workflows/ci.yml

```diff
@@ -16,12 +16,7 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Setup JDK
-        uses: rtyley/setup-java@patch-1
-        with:
-          distribution: corretto
-          java-version: 17 # Currently Java 17 required by `s3mock-testcontainers`
```

Comment:
One question actually: CI is now using 21, as [specified in .tool-versions](https://github.com/guardian/redirect-resolver/blob/f606462701c405100e779207803a21679380e540/.tool-versions). It looks like this is a change: is that ok?

Reply:
Oh my goodness, this is a great point. So actually, it turns out this line, using Java 17 and the comment, is all a copy-n-paste from `etag-caching`, which _does_ [use s3mock](https://github.com/guardian/etag-caching/blob/19542dad7fda70d92675ff27c237f017db6d194f/build.sbt#L35) - but `redirect resolver` doesn't use `s3mock`, so we definitely want to remove this line - and as we've seen, the tests for `redirect resolver` pass with Java 21!

Reply:
Great! I figured it probably worked since there were no failures, but was curious about what had happened.